docker/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM node:16
+# LABEL org.opencontainers.image.authors="Mogztter#Guillaume Grossetie"
+LABEL version="15.4.0"
+LABEL description="This docker image contains \
+asciidoctor-web-pdf to generate websites and pdf from asciidoc(tor)."
+
+ENV NODE_ENV="production"
+
+# Install latest chrome dev package and fonts to support major charsets (Chinese, Japanese, Arabic, Hebrew, Thai and a few others)
+# Note: this installs the necessary libs to make the bundled version of Chromium that Puppeteer
+# installs, work.
+RUN apt-get update \
+    && apt-get install -y wget gnupg \
+    && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+    && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
+    && apt-get update \
+    && apt-get install -y google-chrome-stable fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-khmeros fonts-kacst fonts-freefont-ttf libxss1 \
+      --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /home/pptruser
+
+# Install puppeteer into /home/pptruser/node_modules.
+RUN yarn add "puppeteer@15.4.0" \
+    # Add user so we don't need --no-sandbox.
+    # same layer as npm install to keep re-chowned files from using up several hundred MBs more space
+    && groupadd -r pptruser && useradd -r -g pptruser -G audio,video pptruser \
+    && mkdir -p /home/pptruser/Downloads \
+    && chown -R pptruser:pptruser /home/pptruser \
+    && (node -e "require('child_process').execSync(require('puppeteer').executablePath() + ' --credits', {stdio: 'inherit'})" > THIRD_PARTY_NOTICES)
+
+ENV PUPPETEER_SKIP_DOWNLOAD=true \
+    PUPPETEER_EXECUTABLE_PATH=/usr/bin/google-chrome
+
+WORKDIR /usr/src/app
+
+RUN yarn global add @asciidoctor/core asciidoctor-pdf \
```

Comment:
My idea was to `COPY` the following directories/files and then use `npm i`:

- bin
- css
- fonts 
- lib 
- package*.json

That way we install the current/development version.

When using `RUN yarn global add @asciidoctor/core asciidoctor-pdf` we will install the latest _published_ version (which is nondeterministic)

We should also install a specific version of `@asciidoctor/core@2.2.6` or `@asciidoctor/core@^2.2.0`.

Then we can use an `ENTRYPOINT`:

```
ENTRYPOINT ["node" "./bin/asciidoctor-web-pdf"]
```

```
docker run --rm asciidoctor-web-pdf --version
```

(we don't need to call `asciidoctor-web-pdf`, and the `--version` will be appended to the command)